photon-lib/py/photonlibpy/packet.py

```diff
@@ -0,0 +1,140 @@
+import struct
+from wpimath.geometry import Transform3d, Translation3d, Rotation3d, Quaternion
+import wpilib
+
+class Packet:
+
+    """
+     * Constructs an empty packet.
+     *
+     * @param self.size The self.size of the packet buffer.
+    """
+    def __init__(self, data:list[int]):
```

Comment:
Why is this a `list[int]`? I'd expect this to be `bytes`

Reply:
Inherited from how NT4 returns the raw bytes topic value

Reply:
That sounds like a bug. We've put effort into making sure raw values are returned to Python as `bytes`. Could you please file an issue over on mostrobotpy?